model_compression_toolkit/qat/keras/qat_model_builder.py

```diff
@@ -45,7 +45,9 @@ def _is_qat_applicable(node: common.BaseNode,
         A boolean whether the layer is to be wrapped with a QuantizeWrapper
     """
 
-    return fw_info.is_kernel_op(node.type) and node.is_weights_quantization_enabled()
+    is_weight_quantization = fw_info.is_kernel_op(node.type) and node.is_weights_quantization_enabled()
```

Comment:
I don't think we need fw_info here as it takes the quantization info from the node (which comes from the TPC)